README.md

```diff
@@ -1,53 +1,109 @@
-# Wiki
+# Publish to GitHub wiki action
 
-This action takes the contents of a provided directory and synchronizes it with your
-repository's wiki. This allows, for example, pull requests for the wiki.
+📖 Deploy docs from your source tree to the GitHub wiki
 
-The goal is to expose the wiki to PRs, so that potential contributors who aren't
-collaborators can contribute to the wiki and go through a review process. Alternatively,
-one can disabled the "Restrict editing to collaborators only" setting for their wiki,
-but this allows the community to make unreviewed, potentially unwanted, changes.
+<div align="center">
 
-By default, it will use the `.github/wiki` directory.
+![](https://placekitten.com/600/400)
 
-## Before using this action
+<!-- prettier-ignore -->
+[Awesome GitHub wikis](https://github.com/MyHoneyBadger/awesome-github-wiki#readme)
+| [🆕 What's new in v1](#tbd)
```

Comment:
```suggestion
| [🆕 Release Notes](https://github.com/spenserblack/actions-wiki/releases/latest)
```
I think we can just point to the latest release.

Reply:
👍

Reply:
Actually, now I remember why I did this: to link to a _specific Announcement post_ that would do more than just a release notes would. You know how release notes are more "technical-ish" than a general "here's a 'blog-like' post about what's new" type thing?

Like this for instance: https://devblogs.microsoft.com/typescript/announcing-typescript-4-8/ vs https://github.com/microsoft/TypeScript/releases/tag/v4.8.2

The idea what that _maybe_ for v1 there would be a "here's a short 'post-like' GitHub Discussion (Announcement) about what this new thing is and why it's better than X, Y, or Z"

But yeah, for now linking to release notes instead (probably without the 🆕 emoji) is a good idea

Reply:
If I could rephrase: It's like the emojis _take the place of_ the "bullet" that the screen reader would usually read out, so it doesn't really make sense to have _both_ in there? Especially for such a short list.